.github/ISSUE_TEMPLATE/0-bug-report.md

```diff
@@ -6,9 +6,17 @@ assignees: ''
 
 ---
 
+## Hello!
```

Comment:
@lukehoban Do we need this prelude? It seems this would be common knowledge to anybody interacting with GitHub. Feels like it just adds noise without any value.